app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailsFragment.java

```diff
@@ -110,39 +85,13 @@ public void onDetach() {
 
     @Override
     public void showProgressBar(boolean show) {
-        ViewUtils.showView(progressBar, show);
-    }
-
-    @Override
-    public void showEventName(String name) {
-        tvEventTitle.setText(name);
-    }
-
-    @Override
-    public void showDates(String start, String end) {
-        tvStartDate.setText(start);
-        tvEndDate.setText(end);
+        ViewUtils.showView(binding.progressBar, show);
```

Comment:
Can't we use a boolean binding field to show the progressbar and setting it to `true` `false` accordingly.